neuromation/cli/config.py

```diff
@@ -198,12 +199,28 @@ def config() -> None:
     click.echo(f"You can use docker client with neuro registry: {registry_str}")
 
 
+@command()
+@async_cmd()
+async def get_clusters(root: Root) -> None:
+    """
+    Fetch and display the list of available clusters from the Neuro Platform.
+
+    """
+
+    config = await get_server_config(root.client._session, root.url, root.auth)
+    fmt = ClustersFormatter()
+    pager_maybe(
+        fmt(config.clusters, config.cluster_config.name), root.tty, root.terminal_size
```

Comment:
I think here instead of `config.cluster_config.name` we should use the name of the cluster stored in the user's local config.

Reply:
Makes sense.
The value doesn't exist at the moment though, I'll add it on implementing `select-cluster` command.